recipes/automake/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conan import ConanFile
+from conans import AutoToolsBuildEnvironment, tools
```

Comment:
So all the changes above where for the (pre)upgrade to v2. But this file seems to be written in the old style (like what it was before).

References to AutoToolsBuildEnvironment instead of the other Auto- things, etc. even the same 'DONT COPY' comment for 'extra.m4'.

Reply:
Hence the test_v1_package folder. Which is to ensure that Conan V1 isn't broken for other people